packages/util/src/address.ts

```diff
@@ -139,3 +70,89 @@ export class Address {
     return new Uint8Array(this.bytes)
   }
 }
+
+/**
+ * Returns the zero address.
+ */
+export function createZeroAddress(): Address {
+  return new Address(zeros(20))
+}
+
+/**
+ * Returns an Address object from a bigint address (they are stored as bigints on the stack)
+ * @param value The bigint address
+ */
+export function createAddressFromBigInt(value: bigint): Address {
+  const bytes = bigIntToBytes(value)
+  if (bytes.length > 20) {
+    throw new Error(`Invalid address, too long: ${bytes.length}`)
+  }
+  return new Address(setLengthLeft(bigIntToBytes(value), 20))
```

Comment:
```suggestion
  return new Address(setLengthLeft(bytes, 20))
```

No need to bigint -> bytes twice ;)

Reply:
ah, of course, thanks